src/components/Swap.tsx

```diff
@@ -198,8 +219,8 @@ const Swap = () => {
       <motion.div className="flex justify-between items-center gap-8 " layout>
         <h1 className="text-2xl font-semibold text-slate-800">Stable Swap</h1>
       </motion.div>
-      {!assetsLoaded ? (
-        <CustomLoader text="Waiting for wallet..." />
+      {!areAnchorsLoaded ? (
+        <CustomLoader text="Please connect wallet..." />
```

Comment:
... implies something is to come. 
```suggestion
        <CustomLoader text="Please connect wallet" />
```
though maybe the last one was a better message and the instruction to connect wallet should go somewhere else. a toast?